setup.py

```diff
@@ -35,7 +35,7 @@
       include_package_data=True,
       zip_safe=True,
       install_requires=[
-        "pymongo>=3.0,<3.8",
+        "pymongo>=3.0,<3.12",
```

Comment:
If we only want to allow up to 3.10, I think it should be:
```suggestion
        "pymongo>=3.0,<3.11",
```